### PR TITLE
hard-code reminders with li classes

### PR DESCRIPTION
**Pull request by stephanieea**
## Purpose

The problem is that we need to display reminders on the screen.  
 Closes [Issue #1SeeAllReminders](https://github.com/turingschool-projects/1610-remember-9/issues/1])

Closes Issue#1

## Approach

We added an array of reminders and added the classname to the li tag.  

### Learning

We referenced the ember-groceries lesson from school and the ember docs to ensure we were making changes in the correct places.  

### Open Questions and Pre-Merge TODOs

We hard coded the reminders but weren't sure if your intention was to create a ui so that the user could enter reminders.  

### Test coverage 

We made the first acceptance reminder list test, that was already written, pass.

### CODE REVIEW

File: tests/unit/routes/index-test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
```
Comment (martensonbj) [inline tests/unit/routes/index-test.js]: This is file is not necessary - we can delete it.

File: app/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
```
Comment (martensonbj) [inline app/routes/index.js:1]: This logic should live in a `reminders` route. Generate this with `ember g route reminders` and rethink how we are handing our model to the template. The index route/template should only be used if we need default material on a page, in this issue we want our logic to be associated directly with a `reminders` route. 

Secondly, Mirage is generating 5 reminders for you and popping them into your data store. Once you have the reminders route up created, look into how to find records from the store and pass them to the `reminders.hbs` template.

**Review by martensonbj — CHANGES_REQUESTED** [review]
Few changes - let me know if you have questions on my comments.

File: app/routes/index.js (hunk 2 above)
Comment (martensonbj) [reply app/routes/index.js:1]: AKA you shouldn't have to hard code in any reminders at this point.

File: app/templates/index.hbs [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<h1>REMINDERS</h1>
+
+{{#each model as | reminder |}}
+  <ul><li class='spec-reminder-item'>{{reminder}}</li></ul>
```
Comment (Tman22) [inline app/templates/index.hbs]: The <ul> tag should be outside the {{each}}. Currently you are making 5 <ul> with a single <li> in each.

Comment (martensonbj) [reply app/templates/index.hbs]: As a follow up - this is ok if you want to treat each reminder as a UL< although I think it makes more sense to treat each reminder as an article with a title (say maybe h2), date (p) and notes (p).

**Comment by stephanieea** [comment]
@Tman22 @martensonbj We moved the reminder logic to the reminder route.  We also moved the ul tag outside of the .spec-reminder-item class.  We deleted the unnecessary file.  We added the changes to pull in reminders from mirage. 🎉

File: tests/integration/components/reminder-item-test.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { moduleForComponent, test } from 'ember-qunit';
```
Comment (martensonbj) [inline tests/integration/components/reminder-item-test.js:1]: 👍

File: app/router.js [hunk 5]
```diff
@@ -7,6 +7,10 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+//  this.route('reminders');
+  this.route('reminders', { path: '/' });
 });
```
Comment (martensonbj) [inline app/router.js:12]: Good adjustment! Let's clean up the extra lines and comments here now.

**Review by martensonbj — APPROVED** [review]

**Comment by martensonbj** [comment]
Looks good! Keep an eye on extra lines/spacing in the next PRs

**Pull request merged**